src/EditorFeatures/Core/InlineHints/InlineHintsDataTaggerProvider.cs

```diff
@@ -61,7 +61,10 @@ protected override ITaggerEventSource CreateEventSource(ITextView textViewOpt, I
             return TaggerEventSources.Compose(
                 TaggerEventSources.OnViewSpanChanged(ThreadingContext, textViewOpt, textChangeDelay: TaggerDelay.Short, scrollChangeDelay: TaggerDelay.Immediate),
                 TaggerEventSources.OnWorkspaceChanged(subjectBuffer, TaggerDelay.Immediate, _listener),
-                TaggerEventSources.OnOptionChanged(subjectBuffer, InlineHintsOptions.DisplayAllOverride, TaggerDelay.Immediate),
+                // Ctrl-alt is the initial trigger for a lot of features (like ctrl-alt-p, attach to debugger). So bring
+                // this up not quite immediately so that users don't get flashes of this appearing when running other
+                // commands.
+                TaggerEventSources.OnOptionChanged(subjectBuffer, InlineHintsOptions.DisplayAllOverride, TaggerDelay.NearImmediate),
```

Comment:
💭 We really need this to start calculation on `Immediate` delay, but avoid rendering notification faster than `NearImmediate`. I've seen cases where calculation is not immediate and this delay can be used for latency hiding.